client/src/views/organizations/OrganizationInformationPage.vue

```diff
@@ -1,207 +1,55 @@
-                    fill="clear"
-                    size="small"
-                    class="show-users-button"
-                    v-show="false"
-                  >
-                    {{ $msTranslate('OrganizationPage.infoPage.users.seeUsers') }}
-                  </ion-button>
-                </div>
-                <div class="user-active-list">
-                  <!-- Admin -->
-                  <div class="user-active-list-item">
-                    <tag-profile :profile="UserProfile.Admin" />
-                    <ion-text class="user-active-list-item__value title-h4">
-                      {{ orgInfo.users.admins }}
-                    </ion-text>
-                  </div>
-                  <!-- Standard -->
-                  <div class="user-active-list-item">
-                    <tag-profile :profile="UserProfile.Standard" />
-                    <ion-text class="user-active-list-item__value title-h4">
-                      {{ orgInfo.users.standards }}
-                    </ion-text>
-                  </div>
-                  <!-- Outsiders if allowed -->
-                  <div
-                    v-if="orgInfo.outsidersAllowed"
-                    class="user-active-list-item"
-                  >
-                    <tag-profile :profile="UserProfile.Outsider" />
-                    <ion-text class="user-active-list-item__value title-h4">
-                      {{ orgInfo.users.outsiders }}
-                    </ion-text>
-                  </div>
-                </div>
-              </div>
-
-              <!-- Revoked -->
-              <div class="org-user-list-item user-revoked">
-                <div class="user-revoked-header">
-                  <ion-title class="user-revoked-header__title title-h5">
-                    {{ $msTranslate('OrganizationPage.infoPage.users.revokedUsers') }}
-                  </ion-title>
-                  <span class="title-h4">{{ orgInfo.users.revoked }}</span>
-                  <!-- if we decide to show a modal with the revoked users list -->
-                  <ion-button
-                    fill="clear"
-                    size="small"
-                    class="show-users-button"
-                    v-show="false"
-                  >
-                    {{ $msTranslate('OrganizationPage.infoPage.users.seeUsers') }}
-                  </ion-button>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
+  <ion-page class="organization-page">
+    <ion-content
+      :fullscreen="true"
+      class="organization-page-content"
+    >
+      <!-- ------------- Switch organization ------------- -->
+      <div
+        class="organization-header"
+        @click="openOrganizationChoice($event)"
+        v-if="isSmallDisplay && userInfo"
+      >
+        <ion-avatar class="organization-header-avatar body-lg">
+          <span v-if="!isTrialOrg">{{ userInfo ? userInfo.organizationId.substring(0, 2) : '' }}</span>
```

Comment:
You can remove this ternary, the condition is already checked in the `div` encapsulating it